src/views/vmd-rdv.view.ts

```diff
@@ -212,28 +212,40 @@ export abstract class AbstractVmdRdvView extends LitElement {
 
     async connectedCallback() {
         super.connectedCallback();
+    }
+
+    disconnectedCallback() {
+        super.disconnectedCallback();
+
+        if(this.lieuBackgroundRefreshIntervalId) {
+            clearInterval(this.lieuBackgroundRefreshIntervalId);
+            this.lieuBackgroundRefreshIntervalId = undefined;
+        }
+    }
+
+    launchCheckingUpdates() {
+        console.log("lancement boucle");
+        clearInterval(this.lieuBackgroundRefreshIntervalId);
         this.lieuBackgroundRefreshIntervalId = setDebouncedInterval(async () => {
+            console.log("checking");
             const currentSearch = this.currentSearch
             if(currentSearch) {
                 const codeDepartement = SearchRequest.isByDepartement(currentSearch)
-                  ? currentSearch.departement.code_departement
-                  : currentSearch.commune.codeDepartement
+                    ? currentSearch.departement.code_departement
+                    : currentSearch.commune.codeDepartement
                 const derniereMiseAJour = this.lieuxParDepartementAffiches?.derniereMiseAJour
                 const lieuxAJourPourDepartement = await State.current.lieuxPour(codeDepartement, true)
                 this.miseAJourDisponible = (derniereMiseAJour !== lieuxAJourPourDepartement.derniereMiseAJour);
 
+                // we stop the update check if there has been one
+                if(this.miseAJourDisponible) {
+                    clearInterval(this.lieuBackgroundRefreshIntervalId);
```

Comment:
@Aymerik Attention à ne pas oublier de nullifier le `this.lieuBackgroundRefreshIntervalId` après le `clearInterval` ... ou bien, mieux : de mutualiser le `clearInterval` du `disconnectedCallback()` (qui passe le `this.lieuBackgroundRefreshIntervalId` à `undefined`) avec celui-ci